Source/Blazorise/Components/Modal/Modal.razor

```diff
@@ -5,7 +5,7 @@
         <div @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" role="dialog" @attributes="@Attributes">
             @if ( RenderMode == ModalRenderMode.Default || ( RenderMode == ModalRenderMode.LazyReload && IsVisible ) || ( RenderMode == ModalRenderMode.LazyLoad && lazyLoaded ) )
             {
-                @if ( ShowBackdrop )
+                @if ( ShowBackdrop && IsBackdropVisible )
```

Comment:
Do we need to kill all the `IsBackDropVisible` "Gibberish"? Seems to just be syncing up with the `visible` state, no? Maybe for now it isn't needed?

Reply:
This is a good approach from what I can see in the code. BackdropVisible must be set at certain places so it would work as expected.